app/views/system_admin/upload_fundings/index.html.erb

```diff
@@ -0,0 +1,87 @@
+<% page_title = "Upload funding information" %>
```

Comment:
Why we don't we just stash it in translations?

Reply:
Done